packages/contracts/src/utils/exchange_wrapper.ts

```diff
@@ -225,6 +225,30 @@ export class ExchangeWrapper {
         const filledAmount = new BigNumber(await this._exchange.filled.callAsync(orderHashHex));
         return filledAmount;
     }
+    public async getOrderInfoAsync(
+        signedOrder: SignedOrder,
+    ): Promise<[number /* orderStatus */, string /* orderHash */, BigNumber /* orderTakerAssetAmountFilled */]> {
```

Comment:
Commented above about this contract function returning an OrderInfo struct, if it did it would also be typed in the generated contract wrappers and would bubble up here.